packages/terra-date-picker/tests/wdio/calendar-filter-spec.js

```diff
@@ -0,0 +1,93 @@
+Terra.describeViewports('Calendar Filter', ['tiny'], () => {
+  describe('Calendar-Filter-Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-picker/date-picker/calendar-filter/calendar-filter-default'));
+
+    Terra.it.validatesElement();
```

Comment:
Do we need to add more tests to validate keyboard interactions and accessibiltiy?

Reply:
Calendar-filter have most of its functionalities from date-picker and date-picker has tests for keyboard interactions and accessibiltiy. So I haven't added same tests in calendar-filter as well.